src/nupic/algorithms/Svm.hpp

```diff
@@ -333,7 +325,7 @@ struct decision_function {
  * n_sv = n_sv[n_class], number of SVs for each class
  * probA, probB = [n_class*(n_class-1)/2]
  */
-class svm_model : public Serializable<SvmModelProto> {
+class svm_model {
```

Comment:
OT: TODO investingate, what is the SVM class doing, possibly replace the algo with existing solution (header include)?